.github/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>판다마켓</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="style.css"> 
+
+    </head>
+    <body>
+        <header>
+            <div class = "top">
```

Comment:
```suggestion
            <div class="top">
```
일관적인 포맷팅을 할 수 있도록 습관을 들여볼까요?